ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -284,10 +296,38 @@ static int HandleVnodeOperation(
                 }
             }
         }
+        else if (ActionBitIsSet(action, KAUTH_VNODE_DELETE))
```

Comment:
Will the DELETE bit always be exclusive of all the bits we checked for above? You may need to drop this `else` to avoid missing events.